javascript/mobile-menu.js

```diff
@@ -0,0 +1,17 @@
+const mobileMenu = document.querySelector(".menu--mobile");
+const sidebarBtn = document.querySelector(".sidebar-btn");
```

Comment:
Seems like you have a class naming that does not follow BEM convention.
Makes sense to name it `sidebar__btn`